python/getCustomAttributes.py

```diff
@@ -65,9 +65,9 @@ def getCustomAttributes(data: dict) -> dict:
         "snakeHealth": you["health"],
         "snakeLength": you["length"],
 
-        "snakeGameWinnerName": board["snakes"][0].name if len(board["snakes"]) > 0 else Null,
-        "snakeGameWinnerId":  board["snakes"][0].id if len(board["snakes"]) > 0 else Null,
-        "snakeGameIsWin": board["snakes"][0].name == you["name"] if len(board["snakes"]) > 0 else False,
+        "snakeGameWinnerName": board["snakes"][0]["name"] if len(board["snakes"]) > 0 else None,
```

Comment:
Looks like this bit was a Javascript porting error - there is no `Null` in Python, only `None', also accessing `'name` and 'id` had to be done via dictionary access, as they were not object attributes.

Reply:
100% a porting error. Writing JS all day has eroded my python skills 😩